pom.xml

```diff
@@ -426,6 +426,32 @@
           <serverId>ossrh</serverId>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-checkstyle-plugin</artifactId>
+        <version>${version.checkstyle.plugin}</version>
+        <dependencies>
+          <dependency>
+            <groupId>com.puppycrawl.tools</groupId>
+            <artifactId>checkstyle</artifactId>
+            <version>${version.checkstyle}</version>
```

Comment:
Where is `version.checkstyle` delcared?

Reply:
it is declared only as a property not as a pluginDependency so it's not working without it.

Reply:
I took this code from there: https://github.com/wildfly-swarm/wildfly-swarm/blob/fb119fbdd3ef22e12d7a3f37df12400407f03d1c/pom.xml#L136